docs/sources/flow/concepts/config-language/_index.md

```diff
@@ -1,14 +1,20 @@
 ---
 aliases:
+- /docs/grafana-cloud/agent/flow/concepts/config-language/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/concepts/config-language/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/concepts/config-language/
+- /docs/grafana-cloud/send-data/agent/flow/concepts/config-language/
+- concepts/configuration-language/
+# Previous page aliases for backwards compatibility:
 - /docs/grafana-cloud/agent/flow/config-language/
 - /docs/grafana-cloud/monitor-infrastructure/agent/flow/config-language/
 - /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/config-language/
 - /docs/grafana-cloud/send-data/agent/flow/config-language/
 - configuration-language/
```

Comment:
Relative aliases also need to be updated when the page changes.

Given the old location, I believe this was redirecting /docs/agent/<AGENT_VERSION>/flow/configuration-language/. In order to maintain the redirect it now needs to be:

```suggestion
- ../configuration-language/ # /docs/agent/<AGENT_VERSION>/flow/configuration-language/
```

Reply:
Ah, thanks a lot for the links and spotting this. I've been able to test locally and I think I've fixed all the issues.